client/src/pages/ModalBookingScreen.js

```diff
@@ -17,7 +17,7 @@ import {
 	Dropdown,
 } from "semantic-ui-react";
 
-function ModalBookingScreen({ bookingDate, refreshBooking, desks }) {
+function ModalBookingScreen({ bookingDate, refreshBooking, desks, users }) {
 	const [open, setOpen] = React.useState(false);
 	const [name, setName] = useState("");
```

Comment:
would you mind changing default value of name to null, please? the reason for this is because in other places you setName(null)